src/main.rs

```diff
@@ -106,9 +153,14 @@ fn set_priv_to_machine() {
 
 fn enable_risc_interrupts() {
     unsafe {
-        riscv::register::mie::set_mtimer();
+        // riscv::register::mie::set_mtimer();
+        riscv::register::mie::set_mext();
+        riscv::register::mie::set_sext();
```

Comment:
Not that it matters, but section 3.6 of the manual says:

> The E31 supports RISC‑V user mode, providing two levels of privilege: machine (M) and user(U). U-mode provides a mechanism to isolate application processes from each other and fromtrusted code running in M-mode.

So I don't think we have a "supervisor mode" to worry about.

Reply:
ah yeah true